test/java/seedu/addressbook/commands/FindCommandTest.java

```diff
@@ -0,0 +1,70 @@
+package seedu.addressbook.commands;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.util.TestUtil;
+import seedu.addressbook.util.TypicalPersons;
+
+public class FindCommandTest {
+
+    private AddressBook addressBook;
+    private TypicalPersons td;
+    
+    private final List<ReadOnlyPerson> EMPTY_LIST = Collections.emptyList();
+
+    @Before 
+    public void setUp() {
+        this.addressBook = new TypicalPersons().getTypicalAddressBook();
+        this.td = new TypicalPersons();
+    }
+
+    @Test
+    public void execute() throws IllegalValueException {
+        //same word, same case: matched
+        assertFindCommandBehavior(new String[]{"Amy"}, TestUtil.createList(td.amy));
```

Comment:
`createList()` is redundant, my bad. 

(I was under the impression that `Arrays.asList()` was returning `List<Person>` instead of `List<ReadOnlyPerson>`, because my parameters were all `Person`s. Looks like I was wrong.)